monkey/agent_plugins/exploiters/hadoop/config-schema.json

```diff
@@ -0,0 +1,44 @@
+{
+    "title": "Hadoop/Yarn Exploiter",
+    "description": "Remote code execution on HADOOP server with YARN and default settings. Logic based on https://github.com/vulhub/vulhub/tree/master/hadoop/unauthorized-yarn.",
+    "required": [
+        "exploitable_ports",
+        "request_timeout",
+        "http_download_timeout",
+        "id_string"
+    ],
+    "properties": {
+        "target_ports": {
+            "title": "List of target ports",
+            "description": "A list of ports that Hadoop will try to exploit.",
+            "type": "array",
+            "uniqueItems": true,
+            "items": {
+                "type": "number",
+                "minimum": 0,
+                "maximum": 65535
+            },
+            "default": [8088]
+        },
+        "request_timeout": {
+            "title": "Request timeout",
+            "description": "How long to wait for response from the Hadoop service in seconds.",
+            "type": "number",
+            "minimum": 1,
+            "default": "15"
+        },
+        "http_download_timeout": {
+            "title": "HTTP server download timeout",
+            "description": "How long the http server will be open for downloads in seconds.",
```

Comment:
This needs more explanation. The user will have no idea what HTTP server we're talking about.